### DISCUSSION TITLE
Support of FBQ100C8VEB

### DISCUSSION THREAD

**Discussion by stich86** [revk/xESP32-Faikout#398] [Q&A]
Hi all,

has anyone a unit similar to mine?
I've taken a picture of the board and there is X50A/X35A connector, but before spent moneys on adapter and cable I want to know if somelse has already used Faikin on this internal unit.

Thanks in advance!

**Comment by stich86** [comment]
I can confirm that it's working as other unit. So if you want, can be added to the list of tested devices :)

**Reply by revk** [reply]
That is a WiKi so feel free to edit.

**Reply by stich86** [reply]
done, thanks!
